spec/dependency_bumper_spec.rb

```diff
@@ -93,8 +93,7 @@
       Dir.mktmpdir do |dir|
         g = Git.init(dir)
 
-        file = File.new("#{dir}/temp", 'w')
-        file.write('hello world')
+        IO.write("#{dir}/temp", 'hello world')
```

Comment:
The previous code wasn't closing the file so a file descriptor was possible leaking. `IO.write` will take care of creating the file, writing to it and closing the file so that you don't have to remember to close the file.